python/test_cases/05_cris_edits_unit_type.sql

```diff
@@ -0,0 +1,4 @@
+-- CRIS user updates a unit type
+
+-- Check cris.units and unit_type_id should update from 6 to 1
+UPDATE cris.unit_cris_data SET unit_type_id = 1 WHERE unit_id = 2;
```

Comment:
I found that this didnt update the uniq_unit_types correctly? This update should have made it so uniq unit types were 1,2,4 but it still shows 1,2,4,6.